packages/renderer/src/stores/context.ts

```diff
@@ -20,6 +20,13 @@ import type { Writable } from 'svelte/store';
 import { writable } from 'svelte/store';
 import { ContextUI } from '../lib/context/context';
 
+async function getCurrentContext(): Promise<ContextUI> {
+  return Object.entries(await window.contextCollectAllValues()).reduce((result, [key, value]) => {
+    result.setValue(key, value);
+    return result;
+  }, new ContextUI());
+}
+
```

Comment:
or in Loader.svelte we could also send an event to the backend in case or client refresh, so backend could send again new stuff

in that case, backend will send context-value-updated calls for contexts